about.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - About</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<body>
+  <div class="container">
+    <nav>
+        <img  src="img/lambda-black.png" alt="lambda logo" class="services-img">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="">Products</a>
+        <a href="">Blog</a>
+        <a href="contact.html">Contact</a>
+    </nav>
+
+    <header class="header-about-page">
+    <div class="container-about-page">
+      <img src="/home/tom/Desktop/Lambda School/Sprint-Challenge--User-Interface/img/jumbo-about.png" alt="skyscraper"> 
+     <h1>About</h1>
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+  </header>
+
+<section class="company-info">
+  
+  <div>
+    <img src="img/about-plan.png" alt="strategy">
+    <h1>Strategy</h1> 
+    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+    <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank">Learn More</a>  
+  </div>
+
+<div>
+    <img src="img/about-working.png" alt="strategy">
+    <h1>How We Work</h1>
+    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+    <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank">Learn More</a> 
+  </div>
+
+<div class="bottom-div">
+    <img src="img/about-office.png" alt="strategy">
+    <h1>Places We Work</h1>
+    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+    <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank">Learn More</a> 
+  </div>
+
+<div class="bottom-div">
+    <img src="img/about-meeting.png" alt="strategy">
+    <h1>Collaboration</h1>
+    <p> Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+    <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank">Learn More</a> 
+  </div>
+  
+</section>
+
+<section class="work-together">   
+  <h2> Let's Work Together</h2>
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </section>
+
+        
+    <footer>
+      <contact>
```

Comment:
As in index.html, `contact` is not a valid HTML element. Try a `section` with contact as the class.